pkg/restore.go

```diff
@@ -117,13 +117,14 @@ func NewCmdRestore() *cobra.Command {
 	cmd.Flags().StringVar(&kubeconfigPath, "kubeconfig", kubeconfigPath, "Path to kubeconfig file with authorization information (the master location is set by the master flag).")
 	cmd.Flags().StringVar(&opt.namespace, "namespace", "default", "Namespace of Backup/Restore Session")
 	cmd.Flags().StringVar(&opt.appBindingName, "appbinding", opt.appBindingName, "Name of the app binding")
+	cmd.Flags().StringVar(&opt.storageInfo.name, "secret-name", opt.storageInfo.name, "Name of the storage secret")
+	cmd.Flags().StringVar(&opt.storageInfo.namespace, "secret-namespace", opt.storageInfo.namespace, "Namespace of the storage secret")
```

Comment:
Rename the flags to `storage-secret-name` and `storage-secret-namespace` respectively.